### PR TITLE
Added response validation for postman

### PR DESCRIPTION
**Pull request by moodyfox124**

### CODE REVIEW

File: src/support/helpers/generateRandomData.js [hunk 1]
```diff
@@ -1,11 +1,9 @@
 const randomstring = require('randomstring');
 
-function randomData(amount) {
+function randomData() {
     let result = {};
-    for (let i = 0; i < amount; i++) {
-        result[randomstring.generate(4)] = randomstring.generate(4);
-    }
-
+        result["first"] = randomstring.generate(4);
```
Comment (Vigidoriya) [inline src/support/helpers/generateRandomData.js]: I think that it isn't a good idea to generate specific key names in helper function. This function won't be reusable after that.

File: src/support/helpers/generateRandomData.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 const randomstring = require('randomstring');
 
-function randomData() {
+function randomData(firstKey, secondKey) {
     let result = {};
-        result["first"] = randomstring.generate(4);
-        result["second"] = randomstring.generate(4);
+        result[firstKey] = randomstring.generate(4);
+        result[secondKey] = randomstring.generate(4);
```
Comment (Vigidoriya) [inline src/support/helpers/generateRandomData.js]: That is much better, but if I want 1 or more than two params?

Comment (Vigidoriya) [reply src/support/helpers/generateRandomData.js]: What if I want non-string?

Comment (moodyfox124) [reply src/support/helpers/generateRandomData.js]: we can't generate anything but string

File: src/support/helpers/generateRandomData.js [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
+function generateRandomStringData(key, amountOfData, generatedDataLength) {
     let result = {};
-    for (let i = 0; i < amount; i++) {
-        result[randomstring.generate(4)] = randomstring.generate(4);
+    for (let i = 0; i < amountOfData; i++) {
+        result[`${key}${i + 1}`] = randomstring.generate(generatedDataLength);
     }
-
     return result;
 }
```
Comment (Vigidoriya) [inline src/support/helpers/generateRandomData.js]: Your function named "generateRandomStringData" but it returns an object and accept param "key". It looks strange.

**Pull request merged**